include/library/spdm_lib_config.h

```diff
@@ -76,6 +76,11 @@
 #define LIBSPDM_MAX_CERT_CHAIN_BLOCK_LEN 1024
 #endif
 
+/* Maximum size of a vendor defined response message */
```

Comment:
This probably isn't needed, or can be configurable at runtime. But in general, if the endpoint has space for the message it shouldn't artificially limit its size.